pennylane/ops/qubit/qchem_ops.py

```diff
@@ -464,3 +464,146 @@ def _matrix(cls, *params):
     def adjoint(self):
         (theta,) = self.parameters
         return DoubleExcitationMinus(-theta, wires=self.wires)
+
+
+class OrbitalRotation(Operation):
+    r"""OrbitalRotation(phi, wires)
+    Spin-adapted spatial orbital rotation.
+
+    .. figure:: ../../_static/qchem/orbital_rotation_decomposition_extended.png
+        :align: center
+        :width: 100%
+        :target: javascript:void(0);
+
+    .. figure:: ../../_static/qchem/givens_rotation_decomposition.png
+        :align: center
+        :width: 85%
+        :target: javascript:void(0);
+
+    |
+
+    This operation performs a Givens rotation between the orbital bases in the Jordan-Wigner basis using a pair
+    of parallel Givens gates :math:`G(\phi)`, which are equivalent to :class:`~.SingleExcitation()` operation.
+    More precisely, for two neighbouring spatial orbitals :math:`\{|\Phi_{0}\rangle, |\Phi_{1}\rangle\}`,
+    it performs the following transformation
+
+    .. math::
+        &|\Phi_{0}\rangle = \cos(\phi/2)|\Phi_{0}\rangle - \sin(\phi/2)|\Phi_{1}\rangle\\
+        &|\Phi_{1}\rangle = \cos(\phi/2)|\Phi_{0}\rangle + \sin(\phi/2)|\Phi_{1}\rangle,
+
+    with the same orbital operation applied in the :math:`\alpha` and :math:`\beta` spin orbitals.
+
+    **Details:**
+
+    * Number of wires: 4
+    * Number of parameters: 1
+    * Gradient recipe: The ``OrbitalRotation`` operator satisfies a four-term parameter-shift rule
+      (see Appendix F, https://arxiv.org/abs/2104.05695)
+
+    Args:
+        phi (float): rotation angle :math:`\phi`
+        wires (Sequence[int]): the wires the operation acts on
+
+    **Example**
+
+    .. code-block::
+
+        >>> dev = qml.device('default.qubit', wires=4)
+
+        >>> @qml.qnode(dev)
+        ... def circuit(phi):
+        ...     qml.BasisState([1, 1, 0, 0], wires=[0, 1, 2, 3])
```

Comment:
I think this should be:
`qml.BasisState(np.array([1, 1, 0, 0]), wires=[0, 1, 2, 3])`

Reply:
Yeah, I have fixed it now.